### ISSUE TITLE
aarch64 build issues

### ISSUE THREAD

**Issue by TomAugspurger** [MacPython/pandas-wheels#102]
A few issues with the aarch64 build:

1. We're building a wheel for NumPy 1.14.5. I think the idea is to build against the oldest ABI-compatible NumPy for that Python version, but NumPy doesn't have aarch64 wheels for 1.14.5. Can / should NumPy build wheels? I think not, since IIRC there were some patches in 1.17.x or 1.18.x to get ARM64 support. So should pandas build against a different version of NumPy? Or do we just live with building NumPy wheels too? (cc @mattip if you have thoughts)
2. I don't see any aarch64 wheels at https://anaconda.org/scipy-wheels-nightly/pandas/files yet. @janaknat can you look into that?

**Comment by mattip** [comment]
> So should pandas build against a different version of NumPy? 

Yes, at least until the gap closes. Make you lives easier, since older versions will have problems.

> Or do we just live with building NumPy wheels too? 

No.

**Comment by janaknat** [comment]
@TomAugspurger The files show up at https://anaconda.org/multibuild-wheels-staging/pandas/files on push. The scipy-wheels-nightly is used when the build is initiated by a cron job. Is the nightly builds a cron job?

**Comment by janaknat** [comment]
I'll create a PR to use the latest numpy. Should cut down on build times further. @TomAugspurger Timeline on when the aarch64 wheels will show up in pypi?

**Comment by TomAugspurger** [comment]
I think the nightly (cron job) builds are supposed to go to https://anaconda.org/scipy-wheels-nightly/pandas. Release builds on master should go to https://anaconda.org/multibuild-wheels-staging/pandas.

Looks like wheels were skipped for the recent 1.1.3 release. I suppose that's because it's version is specified in the travis file rather than the azure pipeline yaml file.

So short-term we'll need to manually update the pandas version in `.travis.yaml` file.

Long-term we need to update https://github.com/pandas-dev/pandas-release/blob/70ff1463db0e07bcabf85b1a1ba5b20efd3a01f1/scripts/wheels.sh to also update `.travis.yaml`. Any help here would be appreciated @janaknat.

**Comment by janaknat** [comment]
@TomAugspurger Weird. There is a cron check in .travis.yml. Does the Travis account have a cron job scheduled? I see azure-pipelines.yml has the cron job listed up top.

I will create a PR updating the pandas-version here. I'll also open a PR for the pandas-release.

**Comment by TomAugspurger** [comment]
FYI @janaknat I've uploaded the aarch64 wheels for pandas 1.1.3: https://pypi.org/project/pandas/1.1.3/#files.

If you're able to, an update to https://github.com/pandas-dev/pandas-release/blob/70ff1463db0e07bcabf85b1a1ba5b20efd3a01f1/scripts/wheels.sh would be appreciated, so that we don't forget to bump it in the next release.

I'm not sure what's up with the nightly build. There was a cron job, but it may have been from the old config. I just recreated it.
